js/search/mediaObjectModel.js

```diff
@@ -97,10 +97,10 @@ const VideoObjectModel = Lang.Class({
          * The duration of the video in ISO 8601 format. Defaults to empty
```

Comment:
The shard I got from @ffarfan had the value as an int. I assume it was meant to be in milliseconds?